include/VCF_Tasks.h

```diff
@@ -1,79 +1,97 @@
+ *    a boolean value indicating successful initialization.
+ * 2) Define the "run" function. Name this function run_<taskname>. This function should have a void
+ *    return type and will be called at the specified interval.
+ * 3) Define the task itself. This requires using the TsTask constructor and passing in:
+ *    - Execution interval in microseconds
+ *    - Number of iterations (typically TASK_FOREVER)
+ *    - Pointer to the run function
+ *    - Pointer to the task scheduler
+ *    - Initial enabled state (typically false)
+ *    - Optional pointer to the init function
+ * 4) Enable the task in the setup() function with task_name.enable()
  * 
+ * Example:
+ * 
+ * bool init_example_task() {
+ *     // Setup code here
+ *     return true;
+ * }
+ * 
+ * void run_example_task() {
+ *     // Task execution code here
+ * }
+ * 
+ * TsTask example_task(1000, TASK_FOREVER, &run_example_task, &task_scheduler, false, &init_example_task);
  */
 
 /* -------------------------------------------------- */
-/*                   TASK PRIORITIES                  */
+/*                   TASK INTERVALS                   */
 /* -------------------------------------------------- */
 /*
- * (1-10) - Car-critical functions (watchdog, shutdown circuit)
- * (11-100) - Performance-critical functions (reading pedals, interfacing w/ inverters, etc)
- * (100+) - Telemetry, logging functions (could be idle functions, too)
- */
-
-#ifndef VCF_TASKS
-#define VCF_TASKS
-
-/* From HT_SCHED library */
-#include "ht_sched.hpp"
-
-/* From shared_firmware_types library */
-#include "SharedFirmwareTypes.h"
-
-
-
-/**
- * The read_adc1 task will command adc1 to sample all eight channels, convert the outputs, and
- * store them in structs defined in shared_firmware_types. This function relies on adc_1 being
- * defined in VCFGlobals.h.
- */
-bool init_read_adc1_task(const unsigned long& sysMicros, const HT_TASK::TaskInfo& taskInfo);
-bool run_read_adc1_task(const unsigned long& sysMicros, const HT_TASK::TaskInfo& taskInfo);
-extern HT_TASK::Task read_adc1_task;
-
-
-
-/**
- * NOTE: These channels are UNUSED BY DEFAULT and exist ONLY FOR TESTING. You may edit this
- * manually to add sensors.
- * 
- * The read_adc2 task will command adc2 to sample all eight channels, convert the outputs, and
- * store them in a struct defined in shared_firmware_types. This function relies on adc_2 being
- * defined in VCFGlobals.h.
- */
-bool init_read_adc2_task(const unsigned long& sysMicros, const HT_TASK::TaskInfo& taskInfo);
-bool run_read_adc2_task(const unsigned long& sysMicros, const HT_TASK::TaskInfo& taskInfo);
-extern HT_TASK::Task read_adc2_task;
-
-/**
- * The buzzer_control task will control the buzzer control pin. This function relies on the 
- * buzzer_control pin definition in VCF_Constants.h;
+ * Critical functions: 1-10ms (100-1000Hz)
+ * Performance monitoring: 10-100ms (10-100Hz)
```

Comment:
get rid of this comment, this isn't accurate. the most critical functions won't have specified intervals--they'll just run as fast as possible.